server.go

```diff
@@ -109,6 +123,13 @@ func (s *Server) handlePutMhs(w http.ResponseWriter, r *http.Request) {
 }
 
 func (s *Server) handleGetMh(w http.ResponseWriter, r *http.Request) {
+	start := time.Now()
+	found := false
+
+	defer func() {
+		s.m.GetValueKeysLatency.Record(context.Background(), time.Since(start).Milliseconds(), attribute.Bool("found", found))
```

Comment:
A uniform `status` tag with code instead of custom tag per path like `found` would make the metrics more concise and easier to read.